ONBOARDING.md

```diff
@@ -6,17 +6,17 @@
 
 Assuming the PR is approved
 
-- Create Pull Request to add user to https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/README.md#infrastructure.
+- Create Pull Request to add user to [Infrastructure](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/README.md#infrastructure).
 - Ideally request users public GPG key as well as their public SSH key.
 
 ## GitHub
 
 Add the user to the correct Infrastructure team:
 
-* [@admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure) - Super Users
-* [@infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure) - Core Infra Team
-* [@adoptopenjdk-infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/adoptopenjdk-infrastructure) - Can be assigned Infra Issues
-* [@jenkins-admins](https://github.com/orgs/AdoptOpenJDK/teams/jenkins-admins) - Super users on Jenkins
```

Comment:
This section likely needs a bit of an update now that https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1084 is complete, but I'm happy for that to be left for a separate PR